app/views/secondary_content_links/index.html.erb

```diff
@@ -21,85 +21,101 @@
   action: 'Secondary links'
 %>
 
-<%= render 'shared/steps/form_notice', message: notice, status: "success" %>
-
 <%= render '/step_by_step_pages/nav', step_by_step_page: @step_by_step_page, active:"secondary-links" %>
 
-<div class="row">
-  <div class="col-md-12">
-    <div class="panel panel-default">
-      <div class="panel-heading">
-        <h3 class="panel-title">
-          <span class="glyphicon glyphicon-info-sign" aria-hidden="true"></span>
-          What is secondary content?
-        </h3>
-      </div>
-      <div class="panel-body">
-        <p>Secondary content is content which is not linked to in the step by step, but has the step by step sidebar showing on it.</p>
-        <p>We use it for content which:
-          <ul>
-            <li>helps you to complete a task in the journey, but is not the primary piece of content you need to visit to complete that task</li>
-            <li>is optional and not useful enough to be included in the step by step</li>
-          </ul>
-        </p>
-      </div>
-    </div>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <%= render "components/panel", {
```

Comment:
Oh! This is the panel! It might be worth adding the component nearer to where it's used.
I'm surprised that this doesn't exist already as I'm sure I've seen something similar in html attachments.

I know that govuk_publishing_components has a panel component that is green. I wonder if it's better to update that one, rather than create a new one?

[government-frontend](https://government-frontend.herokuapp.com/component-guide/banner) also has something that seems almost identical to our panel as well.

Reply:
@leenagupte's suggestion sounds good to me. Sounds like a strong case to move government-frontend component to the gem and use it. _Panel_ can be confused with the [existing panel component in the gem](https://components.publishing.service.gov.uk/component-guide/panel).